packages/poppler.rb

```diff
@@ -38,6 +38,9 @@ def self.build
   end
 
   def self.install
+    system "pip install six"
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "pip uninstall --yes six"
```

Comment:
Changes like this will require us to bump the version, especially if the old binaries don't work.

Reply:
six is usually required for x86_64. I think